### PR TITLE
Update start page

### PR DESCRIPTION
**Pull request by Kasper-S-Jensen**
- New layout for front page
- navigation to specific movie page. 
- dto mapping bug fix

### CODE REVIEW

File: src/Web/couch-potatoes-web-app/src/components/FrontPageMovieInfoBoxComponent.tsx [hunk 1]
```diff
@@ -10,25 +20,28 @@ export interface FrontPageMovieInfoBoxComponentProps {
...
-            </Flex>
-            <Text textColor="white" fontSize={{base: 'lg', md: 'xl', lg: '2xl'}}>
-                {new Date(movie?.releaseDate as string).toLocaleDateString()}
-            </Text>
-            <StarRatingComponent name="rating" value={movie?.tmdbScore || 0} starCount={10}/>
-        </VStack></>
+        <>
+
+            <VStack>
+                <Text marginTop="1rem" textColor="white" fontSize={{base: 'xl', md: '2xl', lg: '3xl'}}
```
Comment (michaelbui99) [inline src/Web/couch-potatoes-web-app/src/components/FrontPageMovieInfoBoxComponent.tsx:26]: Pretty sure largest responsive value is '2xl' 
https://chakra-ui.com/docs/styled-system/responsive-styles

File: src/Web/couch-potatoes-web-app/src/pages/MovieDetailsPage.tsx [hunk 2]
```diff
@@ -191,30 +197,32 @@ const MovieDetailsPage = () => {
...
-                                                          themeColor={themeColor}></MovieDetailsBottomInformationbox>
+                                                          themeColor={themeColor}/>
                     </GridItem>
 
                     {/*CAST*/}
                     <GridItem colSpan={5}>
                         <MovieDetailsCastComponent themeColor={themeColor} Background_Temp={Background_Temp}
-                                                   movieCredits={movieCredits}></MovieDetailsCastComponent>
+                                                   movieCredits={movieCredits ??emptyCreditsObject}/>
+                        {/*    TODO: MICMIC*/}
```
Comment (michaelbui99) [inline src/Web/couch-potatoes-web-app/src/pages/MovieDetailsPage.tsx:214]: Still todo?

File: src/Web/couch-potatoes-web-app/src/components/FrontPageCollectionViewComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                        </Tooltip>
+
+
+                    </SwiperSlide>
+                ))}
+
+
+            </Swiper>
+
+
```
Comment (michaelbui99) [inline src/Web/couch-potatoes-web-app/src/components/FrontPageCollectionViewComponent.tsx:69]: Reduce white space

**Review by michaelbui99 — APPROVED** [review]

File: src/Web/couch-potatoes-web-app/src/pages/MovieDetailsPage.tsx (hunk 2 above)
Comment (Kasper-S-Jensen) [reply src/Web/couch-potatoes-web-app/src/pages/MovieDetailsPage.tsx]: removed

**Review by michaelbui99 — APPROVED** [review]
I keep approving, but  the fire returns 🔥🔥🔥🔥🔥🔥🔥

**Pull request merged**